cmd/thv-operator/Taskfile.yml

```diff
@@ -342,3 +341,47 @@ tasks:
       - echo "Keycloak will be available at http://localhost:8080"
       - echo "Use 'task keycloak:get-admin-creds' to get login credentials"
       - kubectl port-forward service/keycloak-dev-service -n keycloak 8080:8080 --kubeconfig kconfig.yaml
+
+  build-operator-image-ubi:
+    desc: Build the operator image
+    vars:
+      COMMIT:
+        sh: git rev-parse --short HEAD || echo "unknown"
+      BUILD_DATE: '{{dateInZone "2006-01-02T15:04:05Z" (now) "UTC"}}'
+    cmds:
+      - >
+        eval "{{.CONTAINER_RUNTIME}} build --load
+        -t ghcr.io/stacklok/toolhive/operator:local-ubi
```

Comment:
I think the non-ubi name is `ko.local/thv-operator:<HASH>`, should we use the same name and tag here? @jhrozek

Reply:
I don't have a strong preference, it depends on how the target will be used.

Reply:
I think it's up to you - if quay needs some elaborate auth schemes to push, you'd be on the hook to maintain the secrets but otherwise I don't care :)

Reply:
Thanks @TomerFi !